migration/20170612-add-library-id-to-cachedfeed.sql

```diff
@@ -0,0 +1,13 @@
+-- Remove the existing index from cachedfeeds
+drop index if exists ix_cachedfeeds_lane_name_type_facets_pagination;
+
+-- Add library_id as a foreign key
+alter table cachedfeeds add column library_id integer;
+alter table cachedfeeds add constraint coveragerecords_collection_id_fkey
```

Comment:
the name of this constraint should be cachedfeeds_library_id_fkey, not coveragerecords, right?